index.template.html

```diff
@@ -6392,8 +6392,10 @@ <h3>Webhook Event Example</h3>
   <section id="json-schema-for-validation" class="appendix informative">
   <h1>JSON Schema for TD Instance Validation</h1>
```

Comment:
Should we also include a link to the TM schema here/in a separate section?

Reply:
Might be a good idea. We have it already at https://w3c.github.io/wot-thing-description/#thing-model-concept but maybe a section somewhere that puts all these kind of informative resources would be nice. We now have JSON Schema based TD and TM validation, SHACL based validation and Typescript Types (at scripting API repo)